soh/soh/Enhancements/enhancementTypes.h

```diff
@@ -0,0 +1,46 @@
+typedef enum {
```

Comment:
Maybe not really related to your changes, but could be potentially rename this entire option to "Chest Appearance Matches Contents"? It feels weird to refer to it as "size" while that's only one of the available options.

Reply:
I usually refer to it as "Chest Style Matches Contents"

Allows it to keep it's same acronym from OOTR

Reply:
ooh, i like style, i'll clean that up

Reply:
That works too

Reply:
https://github.com/HarbourMasters/Shipwright/pull/2976/commits/81e3ebe151a6b93b9222f1d38ee64d0f5ce2add1